src/popover/popover.jsx

```diff
@@ -118,7 +120,7 @@ const Popover = React.createClass({
       position: 'fixed',
       top: anchor.top,
       left: anchor.left,
-      zIndex: 20,
+      zIndex: 0,
```

Comment:
I think that we should have a file with some level of zIndex, somethings like https://github.com/twbs/bootstrap/blob/master/less/variables.less#L272.